source/fundamentals/collations.txt

```diff
@@ -0,0 +1,531 @@
+
+The following code snippet shows how you can create an index on the "name"
+field with the "en_US" locale collation in ascending order:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start createIndex
+   :end-before: end createIndex
+
+To check whether you created the collation successfully, retrieve a list
+of the indexes on that collection as follows:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start listIndexes
+   :end-before: end listIndexes
+
+The output of the code above should contain the following:
+
+.. code-block:: none
+   :copyable: false
+
+   { ...
+     "collation": { "locale": "en_US", ... }
+     ...
+   }
+
+The following code snippet shows an example operation that specifies the
+same collation and is covered by the index we created above:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start indexOperation
+   :end-before: end indexOperation
+
+.. _collations-operation:
+
+Operation
+~~~~~~~~~
+
+You can override the default collation on a collection by passing the
+new collation as a parameter to one of the supported operations. However,
+since the operation does not use an index, the operation may not perform
+as well as one that is covered by an index. For more information on the
+disadvantages of sorting operations not covered by an index, see the server
+manual page on :manual:`Use Indexes to Sort Query Results </tutorial/sort-results-with-indexes/>`.
+
+The following code snippet shows an example query operation that specifies
+the Icelandic ("is") collation that is not covered by the default collation
+we created in the :ref:`Collection section <collations-collection>`:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start customCollationOperation
+   :end-before: end customCollationOperation
+
+Index Types That Do Not Support Collations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While most MongoDB index types support collation, the following types support
+only binary comparison:
+
+- :manual:`text </reference/method/db.collection.createIndex/#options-for-text-indexes>`
+- :manual:`2d </reference/method/db.collection.createIndex/#options-for-2dsphere-indexes>`
+- :manual:`geoHaystack </reference/method/db.collection.createIndex/#options-for-geohaystack-indexes>`
+
+.. _collations-options:
+
+Collation Options
+-----------------
+
+This section covers various collation options and how to specify them to
+further refine the ordering and matching behavior.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 25,75
+
+   * - Collation Option
+     - Description
```

Comment:
I wonder if it's worth adding a row for the method name. The names of the options and the names of the method don't appear to follow a consistent scheme, and my first pass at reading this was to read the section intro, look down at the code, then look up at the table.

Reply:
I've tried adding a column (I think that's what you meant) in another commit, but the output format was difficult to read even after adjusting the column width.  It sounds like that doesn't work for you, so I'd be happy to entertain other suggestions.

Here are some options I've explored in other commits:
- Seealso directives take up too much vertical space which make it difficult to fit the table on a single screen.
- List the method name instead of the collation option: the issue with this is not all the method names are intuitive
- Link to the API documentation from bulleted list of method and omit the description: I think the descriptions I've created are more helpful than those provided in the API reference docs.


I'm not sure what this meant in your comment, could you elaborate and suggest how you might solve the issue?

> my first pass at reading this was to read the section intro, look down at the code, then look up at the table.

Reply:
I might just be being too picky, and that's ok. I read the intro, then looked down at the code, referencing the table. The method names don't align with the names of the options as listed, so I was asking if another column could be added. It sounds like that's not ideal to do, so I'm fine leaving it. The content is good, chasing perfect will just slow it down.

Reply:
Here's an old staged version from a prior commit that shows the additional column https://docs-mongodbcom-staging.corp.mongodb.com/a8ae46a/java/docsworker-xlarge/DOCSP-9574-collations/fundamentals/collations/#collation-options